scripts/gen_openapi.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Is there a reason to switch this to bash?

Reply:
Yeah! I was trying to keep it generic shell. But bash supports the inline regex and `BASH_REMATCH`

Reply:
That's a good reason 👍